### ISSUE TITLE
Add logo to the document

### ISSUE THREAD

**Issue by hyrodium** [AtelierArith/Replay.jl#6]
I have designed the logo of this package with the following in mind.

* Designed with ASCII art, like Julia's REPL
  * referring to [this generator](https://patorjk.com/software/taag/#p=display&f=Big&t=Replay)
  * using [julia-mono font](https://juliamono.netlify.app/)
* Replay.jl includes REPL
  * i.e. `occursin("repl", "replay.jl")`
* The colors are from [julia-logo-graphics](https://github.com/JuliaLang/julia-logo-graphics)

```
 _____            _             
|  __ \          | |            
| |__) |__  _ __ | | __ _ _   _ 
|  _   / _ \ `_ \| |/ _` | | | |
| | \ \  __/ |_) | | (_| | |_| |
|_|  \_\___|  __/|_|\__'_|\__. jl
           | |             __| |
           |_|            |___/ 
```

![image](https://user-images.githubusercontent.com/7488140/142187181-1aa32285-54c1-4271-a777-a9b0cc1f5721.png)

Does this look good? If so, I'll make a PR.

**Comment by terasakisatoshi** [comment]
![image](https://user-images.githubusercontent.com/16760547/142213880-f45dc8cb-7ddf-4e8e-8433-fa9d6bd76e23.png)

**Comment by hyrodium** [comment]
This is solved by #7, so I'll close this issue.
